server/src/main/scala/org/http4s/server/middleware/EntityLimiter.scala

```diff
@@ -2,7 +2,9 @@ package org.http4s
 package server
 package middleware
 
+import cats.data.Kleisli
 import fs2._
+
```

Comment:
Imports should be ordered alphabetically (I know that's true here, but just stating it :)) and should not be grouped into java, scala, scala stdlib.

Reply:
Probably an automatic IJ thing? My commits do this too occasionally and it drives Ross mad :smile:

Reply:
It's the default in IJ.

It can be changed in Settings > Editor > Code Style > Scala > Imports, by removing all lines from the Import Layout box, until only `all other imports` is left.

Reply:
Is there any way we can check in a project style setting for that?

If not, I might be ready to give up and do it the IntelliJ default way.  (Which is a complicated and unappealing default, but not so bad it's worth asking every individual contributor to change it.)

Reply:
I'll have a look